README.md

```diff
@@ -1,270 +1,130 @@
-![info](https://user-images.githubusercontent.com/82102170/224715824-c0c4220e-4f20-4799-8419-f8138de35b7a.png)
-
-### Favorite
-
-#### Friend
-
-![friend](https://user-images.githubusercontent.com/82102170/224716414-5c6720bd-6d38-4e2d-9353-6bfaee47700e.png)
-
-#### World
-
-![world](https://user-images.githubusercontent.com/82102170/224716652-ca54f3d1-449b-43f9-81f3-7bd0833c7d9d.png)
-
-#### Avatar
-
-![avatar](https://user-images.githubusercontent.com/82102170/224717146-37681b38-61ef-4302-8104-212c2161dc12.png)
-
-### FriendLog
-
-![friendlog](https://user-images.githubusercontent.com/82102170/224717793-dbbccdfd-4f89-4597-b38e-8070549b2cf8.png)
-
-### Moderation
+# Getting Started
 
-![moderation](https://user-images.githubusercontent.com/82102170/224718331-87ab30b2-aae1-4bbf-9472-7279af48bd5d.png)
+<div align="center">
 
-### Notification
+Download and run the latest installer (`VRCX_Setup.exe`) from [here](https://github.com/vrcx-team/VRCX/releases/latest).
 
-![notification](https://user-images.githubusercontent.com/82102170/224718963-706e2d99-fc35-41a5-be2c-ff45681ab084.png)
+# Features
 
-### PlayerList
+<div align="left">
 
-![image](https://user-images.githubusercontent.com/82102170/224722231-54de7848-340c-4e9d-adce-30bccb6f2e8c.png)
+- :family: Friend, world, and avatar list management
+  - Manage your friends list, world/group/avatar lists outside of VRChat.
+  - Monitor the world/avatar activity of your friends and check their online status.
+  - Keep track of when you first added them and when you last saw them.
+  - See how much time you've spent together in worlds and how many times.
+- :electric_plug: Automatically launch apps when you start VRChat
+  - You can configure VRCX to launch other apps when you start VRChat.
+  - For example, you could have VRCX launch an OSC app or a voice changer app when VRChat opens up.
+- :floppy_disk: World Persistence
+  - For worlds that support the feature, VRCX can save world settings, save states, inventories, and other data!
+  - **Note**: To use this feature, you must have "Allow Untrusted URLs" enabled in your VRChat settings.
+  - For Developers: [Wiki Page - World Persistence (PWI)](https://github.com/vrcx-team/VRCX/wiki/World-Persistence-(PWI))
+- :mag: Search for avatars, users, worlds, and groups
+- :clipboard: Build a local, unrestricted world favorites list
+- :camera: Store world data in the pictures you take in-game, so you can remember that one world you took those cool pictures in like... 6 months ago!
+- :bell: Monitor/respond to notifications
+  - You can send/receive invites and friend requests from VRCX as well as see the instance info of invites that you receive.
+- :scroll: See stats/players for your current instance
+- :tv: See the links to videos and that are playing in the world you're in, as well as various other logged data.
+- :bar_chart: Better Discord Rich Presence
```

Comment:
```suggestion
- :bar_chart: Improved Discord Rich Presence
```
`Better Discord` could lead to people thinking this would have something to do with [Better Discord](https://betterdiscord.app/)